src/app/lib/utilities/fetchPageData/index.js

```diff
@@ -38,15 +35,12 @@ const handleResponse = async response => {
 
   if (status === STATUS_CODE_OK) {
     const pageData = await response.json();
-    const type = path(['metadata', 'type'], pageData);
-    const processedPageData = await preprocess(
-      pageData,
-      getPreprocessorRules(type),
-    );
+
+    console.log('xxxx', pageData);
```

Comment:
Oops

Reply:
don't worry this will never be merged! just a proof of concept